internal/logger/target/http/http.go

```diff
@@ -422,18 +417,25 @@ func (h *Target) startQueueProcessor(ctx context.Context, mainWorker bool) {
 		}
 
 		if err != nil {
+			if errors.Is(err, context.Canceled) {
+				return
+			}
+
 			h.config.LogOnceIf(
 				context.Background(),
 				fmt.Errorf("unable to send webhook log entry(s) to '%s' err '%w': %d", name, err, count),
 				name,
 			)
 
-			if errors.Is(err, context.Canceled) {
-				return
+			if retryIntvl <= 0 {
+				retryIntvl = 3 * time.Second
 			}
 
-			time.Sleep(3 * time.Second)
-			goto retry
+			time.Sleep(retryIntvl)
+			retries++
+			if retries <= maxRetries {
+				goto retry
+			}
```

Comment:
```suggestion
			if retries <= maxRetries  || maxRetries == 0 {
				goto retry
			}
```
maxRetries 0 - shouldn't it be retrying indefinitely

Reply:
maxRetries == 0 means undefined